### DISCUSSION TITLE
Conflict with other chat plug-ins

### DISCUSSION THREAD

**Discussion by FZDNSZYT** [Zrips/CMI#7405] [Q&A]
When I use truss, the cmi sends the repeating message, which appears in version 1.19.2, cmi 3.5.5, and the version of the trucket is 2.0.0 RC-3-3

**Comment by CloudeLecaw** [comment]
this just means you havebt disabled cmis chat complety. you need to disable it correctly also not use cmi chat related features like
- cmis chatmangement
- hovers in chat
- cmi chat channels,
- cmis color management for meassage,pms,nicknames
- cmis nickname Management
- cmis chat comunication with discordsrv
- cmis chatfilter
- etc.

have you checked you disabled all in config?

whats your FULL output of cmi version?

**Reply by FZDNSZYT** [reply]
I have given up on TrChat, can I add text hover content to CMI chat?

**Reply by FZDNSZYT** [reply]
![image](https://user-images.githubusercontent.com/86947716/204491553-3ce89243-ab4d-4e5f-b6e3-d356ace28312.png)
and the json script

**Reply by FZDNSZYT** [reply]
hello, are you still there? I would like to ask whether CMI chat supports custom floating windows and json scripts.

**Reply by CloudeLecaw** [reply]
well you could try iut but as far as i know cmis chat only supports its own hover and no json scripts

**Reply by FZDNSZYT** [reply]
The hover window cannot be edited, right?

**Reply by CloudeLecaw** [reply]
not direcly.
for chat hover you can design the colors etc in the translation file.
but for the action in chat things you just can edit the preformed Action in the config of cmi.
